pkg/services/ngalert/ngalert.go

```diff
@@ -32,6 +36,12 @@ func (ng *AlertNG) Init() error {
 
 	ng.registerAPIEndpoints()
 
+	ng.schedule = schedule{
+		baseInterval: time.Second,
+		channelMap:   channelMap{definionCh: make(map[int64]definitionCh)},
+		stop:         make(chan int64),
+	}
+	ng.schedule.ctx, ng.schedule.cancelFunc = context.WithCancel(context.Background())
 	return nil
```

Comment:
In the old alert engine we create an errgroup based on the main server Context 

https://github.com/grafana/grafana/blob/master/pkg/services/alerting/engine.go#L61 

So that once the server shuts down nested groups/routines get's shut down, not sure if that is better